api/docs/release.dox

```diff
@@ -173,6 +173,10 @@ Further non-compatibility-affecting changes include:
  - Added opnd_create_immed_double(), opnd_get_immed_double() and
    opnd_is_immed_double() to enable the creation and handling of double
    precision floating-point operands.
+ - Added -reset_at_created_thread_count option to allow triggering reset when
```

Comment:
Just an official reminder to remove the PR #4498 code, to have it listed in the review comments for a "Done" etc. (not that github does a good job of listing which comments are not resolved).

Reply:
Removed #4498 code and updated branch which removed the submitted #4499 from this PR. Resolved those comments too.